src/odb/src/zutil/util.cpp

```diff
@@ -177,10 +187,16 @@ void cutRows(dbBlock* block,
   }
   auto rows = block->getRows();
   const int initial_rows_count = rows.size();
-  const int initial_sites_count
-      = std::accumulate(rows.begin(), rows.end(), 0, [](int sum, dbRow* row) {
-          return sum + row->getSiteCount();
-        });
+  const std::int64_t initial_sites_count = std::accumulate(

```

Comment:
@maliberty I've updated the data type to int64_t, but I still see the overflow in the message INFO-303. Debugging it, I've noticed that the `sum` variable overflows when it reaches the max limit for an `int`. I don't get why it happens, since I'm using int64_t for this variable too.

Reply:
Use int64_t(0), [](int64_t sum, ....